projects/counter/tests/counter.test.ts

```diff
@@ -0,0 +1,85 @@
+import { Cl } from '@stacks/transactions';
+import { expect, test } from "vitest";
+
+const accounts = simnet.getAccounts();
+
+test('get-count returns u0 for principals that never called count-up before', () => {
```

Comment:
The idea of `test` is to write an english sentence like "test('if get-count returns u0 for principals that never called count-up before",.."